docs/src/developers_guide/contributing_pytest_conversions.rst

```diff
@@ -0,0 +1,55 @@
+.. include:: ../common_links.inc
+
+.. _converting_from_unittest:
+
+*******************************************
+Converting From ``unittest`` to ``pytest``
+*******************************************
+
+Conversion Checklist
+--------------------
+.. note::
+    Please bear in mind the following checklist is for general use; there may be
+    some cases which require extra context or thought before implementing these changes.
+
+#. Before making any manual changes, run https://github.com/dannysepler/pytestify
+   on the file. This does a lot of the brunt work for you!
+#. Check for references to :class:`iris.tests.IrisTest`. If a class inherits
+   from this, remove the inheritance. Inheritance is unnecessary for
+   pytest tests, so :class:`iris.tests.IrisTest` has been deprecated
+   and its convenience methods have been moved to the
+   :mod:`iris.tests._shared_utils` module.
+#. Check for references  to ``unittest``. Many of the functions within unittest
+   are also in pytest, so often you can just change where the function is imported
+   from.
+#. Check for references to ``self.assert``. Pytest has a lighter-weight syntax for
+   assertions, e.g. ``assert x == 2`` instead of ``assertEqual(x, 2)``. In the
+   case of custom :class:`~iris.tests.IrisTest` assertions, the majority of these
+   have been replicated in
+   :mod:`iris.tests._shared_utils`, but with snake_case instead of camelCase.
+   Some :class:`iris.tests.IrisTest` assertions have not been converted into
+   :mod:`iris.tests._shared_utils`, as these were deemed easy to achieve via
+   simple ``assert ...`` statements.
+#. Check for references to ``setUp()``. Replace this with ``_setup()`` instead.
+   Ensure that this is decorated with ``@pytest.fixture(autouse=True)``.
+   .. codeblock:: python
+
+      @pytest.fixture(autouse=True)
+      def _setup(self):
+         ...
```

Comment:
This has _almost_ rendered correctly, but clearly still needs some tweaking.

![image](https://github.com/user-attachments/assets/52606a9e-0a8b-4f57-b8ec-a1e879e6dbc0)